auditlog_security/models/auditlog_log.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2025 Therp B.V.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AuditlogLog(models.Model):
+    _inherit = "auditlog.log"
+
+    rule_id = fields.Many2one("auditlog.rule", compute="_compute_rule_id", store=True)
+
+    @api.depends("model_id")
+    def _compute_rule_id(self):
+        for log in self:
+            log.rule_id = self.env["auditlog.rule"].search(
+                [("model_id", "=", log.model_id.id)]
+            )
```

Comment:
do we need this whole file after all?

Reply:
Yeah, basically the rule_id computation can also happen in the line model, so I'll see into removing this